script.js

```diff
@@ -1,18 +1,50 @@
 var scissors = "scissors";
 var paper = "paper";
 var stone = "stone";
+var reversedScissors = "reversed scissors";
+var reversedPaper = "reversed paper";
+var reversedStone = "reversed stone";
```

Comment:
[Recommendation]: When defining this types of "fixed" variables then we should use all caps ie. var STONE = "stone"